Detectors/TPC/calibration/include/TPCCalibration/dEdxHistos.h

```diff
@@ -0,0 +1,82 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file dEdxHistos.h
+/// \brief This file provides the container used for time based dE/dx calibration.
+/// \author Thiago Badaró <thiago.saramela@usp.br>
+
+#ifndef ALICEO2_TPC_DEDXHISTOS_H_
+#define ALICEO2_TPC_DEDXHISTOS_H_
+
+#include <string>
+#include <array>
+#include <gsl/span>
+
+// o2 includes
+#include "TPCCalibration/FastHisto.h"
+#include "DataFormatsTPC/TrackCuts.h"
+
+namespace o2::tpc
+{
+
+// forward declaration
+class TrackTPC;
+
+/// Class that creates dE/dx histograms from a sequence of tracks objects
+class dEdxHistos
+{
+  using Hist = FastHisto<float>;
+
+ public:
+  /// Default constructor
+  dEdxHistos() = default;
+
+  /// Constructor that enable tracks cuts
+  dEdxHistos(TrackCuts cuts, unsigned int nBins);
```

Comment:
```suggestion
  dEdxHistos(const TrackCuts& cuts, unsigned int nBins);
```
below you move, which will 'return' an empty input object. Not necessarily what one would expect I think.